docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /go/src/github.com/canow-co/cheqd-node
 COPY . .
 
 # Make node binary
-RUN make build
+RUN make install && make build
```

Comment:
It rather should be `RUN make build && make install`.

Reply:
Agree, but suggest not to change, because there are changes from upstream